subprojects/degrees-and-programs/source/static/app/js/degreesAndProgramsApp.js

```diff
@@ -110,7 +110,7 @@ var chapman = chapman || {};
 				var form = $(this),
 					target = $(event.target);
 
-				if (target.attr('id').includes('keyword')) {
+				if (target.attr('id').indexOf('keyword') !== -1) {
```

Comment:
Was `includes` not working? I'd expect it to be ok for the browsers we'd support:

- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes